packages/server/lib/plugins/child/run_plugins.js

```diff
@@ -14,6 +14,7 @@ const resolve = require('../../util/resolve')
 const browserLaunch = require('./browser_launch')
```

Comment:
Most of the changes in here was just moving a couple methods around so that the order made a little more sense (to me at least) when I was updating the tests for this. The only thing I added were the lines referencing `_process:cross:origin:callback`.